shims/spark300/src/main/scala/com/nvidia/spark/rapids/shims/spark300/Spark300Shims.scala

```diff
@@ -491,6 +491,7 @@ class Spark300Shims extends SparkShims {
           val pathStr = f.toString
           val matchedSet = replaceMap.keySet.filter(reg => pathStr.startsWith(reg))
           if (matchedSet.size > 1) {
+            // never reach here since replaceMap is a Map
```

Comment:
what does this mean?  map can have size 0, do you mean by this point there will be something in it?

Reply:
I mean the condition of (matchedSet.size>1) will not happen, since there is no replication in replaceMap.keySet.

Reply:
eg. s3:/root->alluxio:/root,s3:/root->alluxio:/bar is allowed, and the last one (s3:/root) will override the previous one

Reply:
ok